src/Resources/CachedContents.php

```
@@ -0,0 +1,86 @@
+        ?string $displayName = null,
+        string|Blob|array|Content|UploadedFile ...$parts,
+    ): MetadataResponse {
+        /** @var ResponseDTO<array{name:string,model:string,displayName:?string,usageMetadata:array<string,mixed>,createTime:string,updateTime:string,expireTime:string}> $response */
+        $response = $this->transporter->request(new CreateRequest(
+            model: $this->parseModel($model),
+            systemInstruction: $systemInstruction,
+            tools: $tools,
+            toolConfig: $toolConfig,
+            ttl: $ttl,
+            displayName: $displayName,
+            parts: $parts,
```

PHPStan (2.1.17)

Parameter $parts of class Gemini\Requests\CachedContents\CreateRequest constructor expects array<int, array<Gemini\Requests\CachedContents\Blob|Gemini\Requests\CachedContents\UploadedFile|string>|Gemini\Data\Content|Gemini\Requests\CachedContents\Blob|Gemini\Requests\CachedContents\UploadedFile|string>, array<int|string, array<Gemini\Data\Blob|Gemini\Data\UploadedFile|string>|Gemini\Data\Blob|Gemini\Data\Content|Gemini\Data\UploadedFile|string> given.


(argument.type)